README.md

```diff
@@ -0,0 +1,29 @@
+# Unikraft official website.
+
+You can find the documentation under https://unikraft.org/docs/
+
+## Build the website locally
+
+You can build the website inside a docker environment.
+For this, you will need to install Docker-CE.
+You can do that by following the instructions [here](https://github.com/docker/docker-install), or simply run:
+```bash
+curl -fsSL https://get.docker.com/ | sh
+```
+
+To then deploy the website, run:
+```bash
+$ make container
+$ make devenv
+
+$ make serve # To be run inside the docker env
+```
+You may need special privileges to run the first 2 commands.
+
+You can also deploy the website locally.
+You will need to install `hugo`.
+You can do it by following the instructions [here](https://gohugo.io/getting-started/installing/).
+```bash
+$ hugo serve
+```
+Please note that this may lead to errors due to different hugo versions or other environment problems.
```

Comment:
Add:

After building and deploying the website, `hugo` provides instructions on accessing and using it:

```
Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
```

Access the http://localhost:1313/ URL in the browser to view the website.